pkgs/development/libraries/gtk/2.x.nix

```diff
@@ -67,6 +74,7 @@ stdenv.mkDerivation rec {
   '' else null;
 
   configureFlags = [
+    "--sysconfdir=/etc"
```

Comment:
Does `installFlags = [ "sysconfdir=${placeholder "out"}/etc" ];` as mentioned in https://github.com/NixOS/nixpkgs/issues/65718 not work?

Reply:
We don't want the sysconfdir to be in `$out`, since that would mean that you still can't manage gtk config declaratively. (which this PR tries to change)

Reply:
It would not be `installFlags` would not affect what is hardcoded in the code, only the installation paths.

Reply:
but we want to change what is hardcoded in the code.

Reply:
That is what `configureFlags` are for.

Reply:
> That is what `configureFlags` are for.

configureFlags is used here

Reply:
Right but `installFlags` could be used instead of the patch.